stdlib/ocaml/generate.mc

```diff
@@ -785,14 +785,14 @@ let ocamlEvalBool = lam ast.
 in
 
 let ocamlEvalChar = lam ast.
-  ocamlEval (wrapOCamlAstInPrint ast (printf "'%c'"))
+  ocamlEval (wrapOCamlAstInPrint ast (ulam_ "x" (app_ (printf ("'%c'")) (int2char_ (var_ "x")))))
```

Comment:
I'm pretty sure the `%c` printf-format only works for ASCII characters, so you'll have to do something more complicated here. I think you can convert the integer to a ustring (of course, you want it to be treated as a unicode character rather than as an actual integer) and then print the resulting string using the `%s` format.